config/webpack/environment.js

```diff
@@ -1,34 +1,25 @@
-// const { environment } = require('@rails/webpacker');
```

Comment:
This was dead code that was in the file before these changes. Just doing some cleanup.